src/ts/components/seekbar.ts

```diff
@@ -48,6 +50,60 @@ export interface SeekBarMarker {
   duration?: number;
 }
 
+enum SeekBarType {
+  Vod,
+  Live,
+  Volume
+}
+
+interface ArrowKeyControls {
+  left: () => void;
+  right: () => void;
+  up: () => void;
+  down: () => void;
+}
+
+const arrowKeyControls = (currentValue: number, set: (number: number) => any, range: {min: number, max: number}): ArrowKeyControls => {
+  const controlValue = Math.floor(currentValue);
+
+  return {
+    left: () => {
+      if (controlValue > range.min) {
+        set(controlValue - 1);
```

Comment:
Who defines those steps? `1` and `5`. Can we extract this and maybe make them configurable. I'm not sure if needed but if you have a loooooong VOD, with 1 and 5 sec steps you can seek for a long time.